docs/templates/tutorials/01-basic-tutorial.md

```diff
@@ -0,0 +1,83 @@
+# 01. Basic Tutorial
+
+In Basic Tutorial, you can learn the following things:
+
+* Define Search Space
+* Optimize Objective Function
+
+This tutorial describes how to optimize Hyperparameters using HyperOpt without having a mathematical understanding of any algorithm implemented in HyperOpt.
+
+
+```python
+# Import HyperOpt Library
+from hyperopt import tpe, hp, fmin
+```
+
+Declares a purpose function to optimize. In this tutorial, we will optimize a simple function called `objective`, which is a simple quadratic function.
+
+$$ y = (x-3)^2 + 2 $$
+
+
+```python
+objective = lambda x: (x-3)**2 + 2
+```
+
+Now, let's visualize this objective function.
+
+
+```python
+import matplotlib.pyplot as plt
+import numpy as np
+
+x = np.linspace(-10, 10, 100)
+y = objective(x)
+
+fig = plt.figure()
+plt.plot(x, y)
+plt.show()
+```
+
+
+![png](01-basic-tutorial_files/01-basic-tutorial_6_0.png)
+
+
+We are trying to optimize the `objective` by changing the HyperParameter $x$. That's why we will declare a search space for $x$. The functions related to the search space are implemented in `hyperopt.hp`. The list is as follows.
+
+* `hp.randint(label, upper)` or `hp.randint(label, low, high)`
+* `hp.uniform(label, low, high)`
+* `hp.loguniform(label, low, high)`    
+* `hp.normal(label, mu, sigma)`
+* `hp.lognormal(label, mu, sigma)`
+* `hp.quniform(label, low, high, q)`
+* `hp.qloguniform(label, low, high, q)`
+* `hp.qnormal(label, mu, sigma, q)`
+* `hp.qlognormal(label, mu, sigma, q)`
+* `hp.choice(label, list)`
+
+We will use the most basic `hp.uniform` in this tutorial.
+    
+
+
+```python
+# Define the search space of x between -10 and 10.
```

Comment:
Maybe a comment to say that uniform means that all continuous values have the same probability to be sampled